docs/source/developer/bevel_fabric_minikube_setup.md

```diff
@@ -0,0 +1,168 @@
+[//]: # (##############################################################################################)
+[//]: # (Copyright Accenture. All Rights Reserved.)
+[//]: # (SPDX-License-Identifier: Apache-2.0)
+[//]: # (##############################################################################################)
+
+# Running Bevel DLT hyperledger fabric network on minikube
+
+## Pre-requisites
+
+Before proceeding, first make sure that you've completed [Developer Pre-requisites](./dev_prereq.md).
+
+## Setup minikube
+
+1. Install minikube using instruction [here](https://minikube.sigs.k8s.io/docs/start/) on VM
+
+2. Start minikube. The setup has been tested on a Ubuntu 20.04 VM with atleast 16 GB RAM, 8 vcpu and requires a public ip  
+   ```bash
+   minikube start --memory 12000 --cpus 4 --kubernetes-version=1.19.1 --apiserver-ips=<specify public ip of VM>
+   ```
+3. Start a proxy which is required for ansible controller(to be created later in following steps) to access the minikube k8s 
+   ```bash
+   docker run -d --network minikube -p 18443:18443 chevdor/nginx-minikube-proxy
+   ```
+## Clone forked repo
+
+1. If you have not already done so, fork [bevel](https://github.com/hyperledger/bevel) and clone the forked repo to your machine.
+   ```bash
+   git clone git@github.com:<githubuser>/bevel.git   
+   ```
+
+## Update kubeconfig file
+
+1. Create a `build` folder inside your Bevel repository:
+   ```bash
+   cd ~/bevel
+   mkdir build
+   ```
+2. Copy ca.crt, client.key, client.crt from `~/.minikube` to build:
+   ```bash
+   cp ~/.minikube/ca.crt build/
+   cp ~/.minikube/profiles/minikube/client.key build/
+   cp ~/.minikube/profiles/minikube/client.crt build/
+   ```
+3. Copy `~/.kube/config` file to build:
+   ```bash
+   cp ~/.kube/config build/
+   ```   
+4. Open the above config file in build directory and update file path for certificate-authority, client-certificate and client-key  to point to the respective files copied in build directory.
+
+   ![](./../_static/minikube-config.jpg)
+
+   Update config `server` value to following:
+   ```bash
+   server: https://<specify public ip of VM>:18443
+   ```
+
+   ***   
+   **NOTE**: If you ever delete and recreate minikube, the above steps have to be repeated.
+   ***
+
+## Setup Hashicorp vault
+
+   1. There are two options to setup vault:
+   Setup hashicorp vault and run in dev mode. Install the vault following instructions at following [url]( 
+   https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install)
+   
+      ```bash
+      vault server -dev -dev-root-token-id root -dev-listen-address 0.0.0.0:8200
+      ```
+   OR
+
+   2. One can also use helm charts for vault and do the operations as mentioned in the Workshop series [here](    
```

Comment:
Can you pinpoint the time around which one should start looking for the vault specifics?